README.md

```diff
@@ -1,91 +1,134 @@
-# EVM Placeholder Proof System Verifier 
+# EVM Placeholder proof system verifier 
 
 [![Discord](https://img.shields.io/discord/969303013749579846.svg?logo=discord&style=flat-square)](https://discord.gg/KmTAEjbmM3)
 [![Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=flat-square&logo=telegram&logoColor=dark)](https://t.me/nilfoundation)
 [![Twitter](https://img.shields.io/twitter/follow/nil_foundation)](https://twitter.com/nil_foundation)
 
-This repository contains the smart contracts for validating zero knowledge proofs 
-generated in placeholder proof system in EVM. 
+An application for in-EVM validation of zero-knowledge proofs generated
+with
+the [Placeholder proof system](https://nil.foundation/blog/post/placeholder-proofsystem).
 
 ## Dependencies
 
 - [Hardhat](https://hardhat.org/)
-- [nodejs](https://nodejs.org/en/) >= 16.0
+- [Node.js](https://nodejs.org/) - Hardhat requires an LTS version of Node.js; as of September 2023 it's v18
 
+## Installation
 
-## Clone
-```
+Clone the project from GitHub:
+
+```bash
 git clone git@github.com:NilFoundation/evm-placeholder-verification.git
+```
+
+Or install the package via `npm` from the command line:
+
+```bash
+npm install @nilfoundation/evm-placeholder-verification@1.1.1
+```
+
+or add it to the `package.json`:
+
+```bash
```

Comment:
It's not bash; it's a fragment of JSON. If it doesn't parse well as JSON, just make it plain `text`.